docs/README.md

```diff
@@ -157,12 +159,23 @@ Check out [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-dat
 
 ### Configuring Lookback
 
-The lookback strategy determines how the agent handles existing files on startup. This strategy is determined by `LOGDNA_LOOKBACK`. The set of valid values for this option are:
+The lookback strategy determines how the agent handles existing files on startup. This strategy is determined by `LOGDNA_LOOKBACK`.
+
+The valid values for this option are:
+   * When set to **`none`**: lookback is disabled, and LogDNA continues to read from where the file ended at the time of the stop/restart action. 
+   * When set to **`smallfiles`** (default):
+       * If there is information in the “state file”, use the last recorded state. 
+       * If the file is not present in the “state file” and the file is less than 8KiB, start at the beginning. If the file is larger than 8KiB, start at the end. 
+   * When set to **`start`**:
```

Comment:
hm... in the github markdown viewer, this is shown as `start: *` ...

Reply:
fixedit,thanks

Reply:
There's still an issue with indentation of the list, that is shown in the viewer:


<img width="580" alt="Screenshot 2021-03-31 at 09 40 27" src="https://user-images.githubusercontent.com/2931196/113108490-55636080-9205-11eb-8f2f-8386cd4c93e7.png">